src/style.css

```diff
@@ -1 +1,74 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+
+}
```

Comment:
Do not apply any rules to the `*` selector except `box-sizing`

Reply:
added the file reset css